packages/cli/test/mocks/deployment.ts

```diff
@@ -170,6 +170,16 @@ function setupDeploymentEndpoints(): void {
     res.json({ builds });
   });
 
+  client.scenario.get('/now/deployments/:id/aliases', (req, res) => {
+    const limit = parseInt(req.query.limit);
+    const { id } = req.params;
+    const deployment = deployments.get(id);
+    res.json({
+      aliases: deployment?.alias ? [deployment.alias] : [],
```

Comment:
I believe `deployment.alias` is a string. The `aliases` return value is an array of objects:

```
{
  uid: string;
  alias: string;
  created: string;
  redirect?: string | null;
  protectionBypass?: {
    [key: string]:
      | { createdAt: number; createdBy: string; scope: 'sharable-link' | 'automation-bypass'; }
      | { createdAt: number; lastUpdatedAt: number; lastUpdatedBy: string; access: 'requested' | 'granted'; scope: 'user'; }
  }
}[]
```